draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -4911,6 +4911,11 @@ CONNECT server.example.com:80 HTTP/1.1
 Host: server.example.com:80
```

Comment:
Up there^^ it says:

> A client sending a CONNECT request &MUST; send the authority component (described in Section 3.2 of <xref target="RFC3986"/>) as the request target;

That's not what happens, because the authority component makes the port optional (and it allows userinfo).  I think that you need to make that text correct too.  You can't rely on id est to define normative behaviour.